src/settings/binding.rs

```diff
@@ -0,0 +1,14 @@
+use serde::Serialize;
+
+#[derive(Serialize, Debug)]
+#[serde(tag = "type")]
+pub enum Binding {
+    #[allow(non_camel_case_types)]
+    wasm_module { name: String, part: String },
+}
+
+impl Binding {
+    pub fn new_wasm_module(name: String, part: String) -> Binding {
```

Comment:
this seems odd; why would we want a wasm-specific method on the binding trait? or am i reading this wrong?

Reply:
The idea is too add for each binding type a method (like a factory, constructor), see https://github.com/cloudflare/wrangler/pull/215/files#diff-deeb786858380f08f8f4d40680620a9a if it makes more sense.

Reply:
ok that makes sense, thank you.